portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,71 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <!-- Style -->
+    <style> 
+
+        /* Background Color */
+        body {
```

Comment:
You can add these styles to the `portfolio/src/main/webapp/style.css` file so the css and html are split into separate files

Reply:
Once I copy my style code to the css file, the style doesn't show up on my server for some reason. I just asked my team for help.

Reply:
Figured it out,

Reply:
.*